README.md

```diff
@@ -441,10 +440,9 @@ Discourage any attempt to use external maintenance contractors by peppering your
 This stuff is too clever for the dullard at Software Services Inc., who would
 probably use 50 times as memory & time using the dumb routines in <math.h>.
 */
-class clever_SSInc
-    {
+class clever_SSInc {
     .. .
-    }
+}
```

Comment:
This looks like an obvious typo. 👍

Reply:
Not sure if you are requesting a change here :/

Reply:
Nope.